testutil/ciao-down/prepare.go

```diff
@@ -224,6 +225,11 @@ func prepareEnv(ctx context.Context) (*workspace, error) {
 		ws.UIPath = string(data)
 	}
 
+	data, err = ioutil.ReadFile(path.Join(ws.instanceDir, "vmtype.txt"))
```

Comment:
If we merge this as is, this will break existing ciao-vms.  When they are stopped and restarted they webui port will not be mapped as the vmtype.txt will not exist.